# Your custom GridNode message
float64 x
float64 y
float64 z
bool is_occupied
bool is_inspected
bool need_to_inspect